.github/readme-de_de.md

```diff
@@ -0,0 +1,383 @@
+
+Wir haben eine [Dokumentationswebsite](https://docs.sillytavern.app/), um die meisten deiner Fragen zu beantworten und dir den Einstieg zu erleichtern.
+
+## Was ist SillyTavern?
+
+SillyTavern (oder ST für kurz) ist eine lokal installierte Benutzeroberfläche, die es dir ermöglicht, mit Textgenerations-LLMs, Bildgenerierungsmaschinen und TTS-Sprachmodellen zu interagieren.
+
+Angefangen im Februar 2023 als Fork von TavernAI 1.2.8 hat SillyTavern nun über 100 Mitwirkende und 2 Jahre unabhängiger Entwicklung hinter sich und dient weiterhin als führende Software für versierte KI-Hobbyisten.
+
+## Unsere Vision
+
+1. Wir möchten die Nutzer mit so viel Nutzen und Kontrolle über ihre LLM-Eingabeaufforderungen wie möglich ausstatten. Die steile Lernkurve ist Teil des Spaßes!
+2. Wir bieten keine Online- oder gehosteten Dienste an, noch verfolgen wir programmgesteuert Benutzerdaten.
+3. SillyTavern ist ein Herzensprojekt, das von einer engagierten Community von LLM-Enthusiasten unterstützt wird, und wird immer kostenlos und Open Source sein.
+
+## Zweige
+
+SillyTavern wird mit einem Zweigsystem entwickelt, um ein reibungsloses Erlebnis für alle Nutzer zu gewährleisten.
+
+* `release` -🌟 **Empfohlen für die meisten Nutzer.** Dies ist der stabilste und empfohlene Zweig, der nur aktualisiert wird, wenn wichtige Versionen veröffentlicht werden. Er ist für die Mehrheit der Nutzer geeignet. Typischerweise einmal im Monat aktualisiert.
+* `staging` - ⚠️ **Nicht für den gelegentlichen Gebrauch empfohlen.** Dieser Zweig enthält die neuesten Funktionen, kann jedoch jederzeit instabil sein. Nur für Power-User und Enthusiasten. Mehrmals täglich aktualisiert.
+
+Wenn du nicht vertraut mit der Verwendung der git CLI bist oder nicht verstehst, was ein Zweig ist, mach dir keine Sorgen! Der Release-Zweig ist immer die bevorzugte Option für dich.
+
+## Was brauche ich zusätzlich zu SillyTavern?
+
+Da SillyTavern nur eine Benutzeroberfläche ist, benötigst du Zugriff auf ein LLM-Backend, um Inferenz bereitzustellen. Du kannst AI Horde für sofortiges Chatten ohne weitere Einrichtung verwenden. Darüber hinaus unterstützen wir viele andere lokale und cloudbasierte LLM-Backends: OpenAI-kompatible API, KoboldAI, Tabby und viele mehr. Du kannst mehr über unsere unterstützten APIs in [der FAQ](https://docs.sillytavern.app/usage/api-connections/) lesen.
+
+### Brauche ich einen leistungsstarken PC, um SillyTavern auszuführen?
+
+Die Hardwareanforderungen sind minimal: Es läuft auf allem, was NodeJS 18 oder höher ausführen kann. Wenn du LLM-Inferenz auf deinem lokalen Rechner durchführen möchtest, empfehlen wir eine NVIDIA-Grafikkarte der 3000er-Serie mit mindestens 6 GB VRAM. Überprüfe die Dokumentation deines Backends für weitere Einzelheiten.
+
+### Vorgeschlagene Backends (nicht verbunden)
```

Comment:
lol
```suggestion
### Vorgeschlagene Backends (keine Partnerschaft oder Werbebeziehung)
```

Reply:
The original intent of that heading was not about connecting to the backends. It‘s affiliate.

Heißt wir listen hier mögliche Backends auf, aber wir arbeiten nicht mit denen Zusammen oder kriegen irgendwas dafür die hier zu listen. Wieder so eine schöne Stelle, wo es kein passendes deutsches Wort gibt. Aber „verbunden“ ist es nicht.